File: OPENAPI-REST-API/swagger-client/dart/lib/model/website_extra_info_tables.dart
part of swagger.api;

class WebsiteExtraInfoTables {
  
  WebsiteTable links = null;

  WebsiteTable preview = null;

  WebsiteTable dns = null;

  WebsiteExtraInfoTables();

  @override
  String toString() {
    return 'WebsiteExtraInfoTables[links=$links, preview=$preview, dns=$dns, ]';
  }

  WebsiteExtraInfoTables.fromJson(Map<String, dynamic> json) {
    if (json == null) return;
    links = new WebsiteTable.fromJson(json['links']);
    preview = new WebsiteTable.fromJson(json['preview']);
    dns = new WebsiteTable.fromJson(json['dns']);
  }

  Map<String, dynamic> toJson() {
    return {
      'links': links,
      'preview': preview,
      'dns': dns
     };
  }

  static List<WebsiteExtraInfoTables> listFromJson(List<dynamic> json) {
    return json == null ? new List<WebsiteExtraInfoTables>() : json.map((value) => new WebsiteExtraInfoTables.fromJson(value)).toList();
  }

  static Map<String, WebsiteExtraInfoTables> mapFromJson(Map<String, Map<String, dynamic>> json) {
    var map = new Map<String, WebsiteExtraInfoTables>();
    if (json != null && json.length > 0) {
      json.forEach((String key, Map<String, dynamic> value) => map[key] = new WebsiteExtraInfoTables.fromJson(value));
    }
    return map;
  }
}

File: OPENAPI-REST-API/swagger-client/csharp-dotnet2/src/main/CsharpDotNet2/myadmin-client-csharp-dotnet2/Model/WebsiteExtraInfoTables.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;
using Newtonsoft.Json;

namespace IO.Swagger.Model {

  /// <summary>
  /// Supplementary information tables displayed for a webhosting service (links, DNS, preview).
  /// </summary>
  [DataContract]
  public class WebsiteExtraInfoTables {
    /// <summary>
    /// Gets or Sets Links
    /// </summary>
    [DataMember(Name="links", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "links")]
    public WebsiteTable Links { get; set; }

    /// <summary>
    /// Gets or Sets Preview
    /// </summary>
    [DataMember(Name="preview", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "preview")]
    public WebsiteTable Preview { get; set; }

    /// <summary>
    /// Gets or Sets Dns
    /// </summary>
    [DataMember(Name="dns", EmitDefaultValue=false)]
    [JsonProperty(PropertyName = "dns")]
    public WebsiteTable Dns { get; set; }


    /// <summary>
    /// Get the string presentation of the object
    /// </summary>
    /// <returns>String presentation of the object</returns>
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class WebsiteExtraInfoTables {\n");
      sb.Append("  Links: ").Append(Links).Append("\n");
      sb.Append("  Preview: ").Append(Preview).Append("\n");
      sb.Append("  Dns: ").Append(Dns).Append("\n");
      sb.Append("}\n");
      return sb.ToString();
    }

    /// <summary>
    /// Get the JSON string presentation of the object
    /// </summary>
    /// <returns>JSON string presentation of the object</returns>
    public string ToJson() {
      return JsonConvert.SerializeObject(this, Formatting.Indented);
    }

}
}
